src/Service/Auth0Service.php

```diff
@@ -557,4 +557,23 @@ public function getSalesforceId(string $auth0Id) {
     return ($auth0User['app_metadata']['salesforce_id'] ?? '');
   }
 
+  /**
+   * Create job to send verification email.
+   *
+   * @param string $auth0Id
+   *   Auth0 ID.
+   *
+   * @return bool
+   *   Return true if successful, false if issues.
+   */
+  public function createSendVerificationEmail(string $auth0Id): bool {
+    $response = $this->getManager()->management()->jobs()->createSendVerificationEmail($auth0Id);
+    if ($response->getStatusCode() == 201) {
```

Comment:
Can this be simplified to the following?

```php
return $response->getStatusCode() === 201
```

Reply:
Sure!